packages/fast-components-react-msft/src/action-trigger/action-trigger.schema.json

```diff
@@ -28,12 +29,14 @@
     "reactProperties": {
         "children": {
             "title": "Children",
-            "type": "children"
+            "type": "children",
+            "formPluginId": "@microsoft/fast-components-react-msft/action-trigger/children"
         },
         "glyph": {
             "title": "Glyph",
             "type": "children",
-            "pluginId": "@microsoft/fast-components-react-msft/action-trigger/glyph"
+            "pluginId": "@microsoft/fast-components-react-msft/action-trigger/glyph",
```

Comment:
Is the pluginId here different then the new formPluginId?

Reply:
It is, the `pluginId` is for the data mapping and allow us to manipulate data blobs, the `formPluginId` is for manipulating the schema that goes into the form generator.